book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def intialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  # method for rental
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] A book has many rentals, Implementing the method for `add_rental` let us pass the following parameters ``def add_rental(person, date)``

Reply:
Thanks for the review and reminder of what i left out. the required changes have been made